### ISSUE TITLE
Firebase user creation field values interfere between two sign up form pages

### ISSUE THREAD

**Issue by AbdelrahmanSalehRewaished** [FlutterFlow/flutterflow-issues#2606]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have two sign-up pages having a form element in which both have an email, password, and name text fields
when setting the action for each form button in the two pages trying to create a user document from the Firebase authentication and Create account and trying to set the display_name field (or any other user field), the value of the name field overrides the one in the other page's action, 

1) First Sign up page
![Capture](https://github.com/FlutterFlow/flutterflow-issues/assets/146881379/11beef8c-f4aa-4946-b600-0f412b83b026)

2) Second Sign page
![Capture](https://github.com/FlutterFlow/flutterflow-issues/assets/146881379/b30cb1c5-c51e-4744-b80a-cdd2c3bbe8f9)

causing the following error:

![Capture](https://github.com/FlutterFlow/flutterflow-issues/assets/146881379/f0325154-505c-43aa-9a66-3605ba5e5149)

### Expected Behavior

The normal expected behavior is to have both sign-up forms operating independently, setting a user field here does not affect the other one. 

I doubted that it was because of duplicating the page initially or because of a button copy
but I created the two pages from scratch and without any copying or pasting but still got the same weird and unexpected problem.

I am really disappointed with this!

### Steps to Reproduce

1. Create a blank project
2. Set up Firebase and enable authentication (Email provider) and download configurations file (the normal firebase authentication setting).
3. Create two different signup pages having the following text fields with for example an email, name, password, and confirm password.
4. try creating an on tap action on each page "Sign Up" Button.
5. Choose Backend -> Firebase Authentication -> Create Account 
6. Auth Provider: Email, Email: emailField, Password: passwordField, Confirm Password: confirmPasswordField
7. Set "Create User Document" 
8. Try Setting fields "email" & "display_name" for the Firebase user collection
9. Whenever you set a value in the email here it gets reflected on the other page create user action causing the referenced error.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4KkMrqw5Ngobxc+KrUbO9VlSQsJj8kRYI0lu1TQgAaJbatG7YAW/SkQE9WZ7yhdH03MFqZ+XMdw/CMkvbIN8A6HEmVR4hzyKVqZQLwZGSgR7rUPru8WmtSEtpKC2aY1MOKkSZSIs12dCUB2E2DfPK9fHTXQr7JIUg3JOaiOt/4s3/uQ0SLb3kNh1JWeC/v

### Context

Multi User roles Sign up and authentication (not something fancy)

### Visual documentation

![Capture](https://github.com/FlutterFlow/flutterflow-issues/assets/146881379/3bc0387f-0fb6-4d35-aa4f-4955c3c34c8e)

### Environment

```markdown
- FlutterFlow version: 4.1
- Platform: (General, the error is preventing running or testing)
- Browser name and version: Edge (latest version)
- Operating system and version affected: (Windows 10)
```

**Comment by ignalauret** [comment]
Hey @AbdelrahmanSalehRewaished thanks for your report. I was able to replicate your issue with having 2 create account buttons. Our technical team is going to review it and keep you updated on this thread. <br>Have a great day!

**Comment by dunkah63** [comment]
I see that this issue has been happening for over a year. I have this same problem today. I'm not familiar with coding, I paid for a program to build apps with a group of coders to help build an app to put on the App Store. They haven't seen this issue before. They have video step-by-step instructions and once I got to the part to click the Firebase link in the Tree under the "App Assets" tab as in the video the tab was NOT there. I watched YouTube videos and learned to place the word "firebase" into the search bar but it did not provide all elements that where in the instruction videos.  So I did more research to then get the Authentications page to show up but it was missing some key elements as well. So could someone provide a solution as to how to past this stage in plain English or in a way I can present to my coding team.
